src/python/blazingmq/dev/it/tweaks/generated.py

```diff
@@ -407,6 +407,12 @@ def __call__(
             dispatcher_config = DispatcherConfig()
 
             class Stats(metaclass=TweakMetaclass):
+                class AppIdTagDomains(metaclass=TweakMetaclass):
+                    def __call__(self, value: None) -> Callable:
+                        ...
```

Comment:
Does `...` just mean `pass` here?

Reply:
Yes, `ellipsis` literal does the same nothing as `pass` in this context